### ISSUE TITLE
[TC] [Bug] OP warrior

### ISSUE THREAD

**Issue by Grimfeather** [trickerer/Trinity-Bots#567]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that my issue may get **closed without notice** if I *intentionally* remove or skip any mandatory\* field

### Current behaviour

Bots are extremely tanky, especially warrior bots.
And the warrior bot does too much damage.

I recorded this to showcase what I mean.
https://youtu.be/IiKsELSpBo0

I'm using default settings,
NpcBot.Mult.Damage.Warrior           = 1.0
the warrior in the video is equiped with dungeon blues and he's specced into protection

I've seen him crit with revenge for 6k+ damage. at level 60.
that was on lower level enemies tbf, in Uldaman.
but still his revenge is critting for 3.2k in the linked video.

Some questions
- was this done on purpose perhaps to help with aggro for the warrior?
- are bots perhaps tweaked for lvl80 wotlk end game content and being OP at early stages is a by product?
- is there any way for me to tweak these settings myself? I understand I can lower his damage with the multiplier setting. but that still leaves his tankiness.
- are they popping potions non stop? in the video at about the 1 minute mark he goes from low health to full health suddenly, I can't tell how he does that, it's not in the combat log.

It just feels wrong, you know. And I want it to feel right.

### Expected behaviour

I expected the warrior to die of course. Being able to solo lbrs at lvl 60 shouldn't be possible.

### Steps to reproduce the problem

kinda obvious, but
- level 60
- give the warrior bot some blue dungeon gear, protection spec
- .tele lbrs
- watch the show.

### TrinityCore or AzerothCore

TrinityCore

### Core rev. hash/commit

TrinityCore rev. 4bc06c24fc4a 2023-06-02 21:26:47 +0700 (npcbots_3.3.5 branch) (Win64, RelWithDebInfo, Static) (worldserver-daemon)

### Operating system

Win 10

**Comment by Russianranger** [comment]
Some critical questions here;
Dungeon blues - were these "classic" dungeon blues? Or "TBC" dungeon blues? 
Also - that modifier is one of 3 different modifiers - there are modifiers for level brackets, and personally owned bots overall damage. 1.0 is scaled for WotLK content, but not necessarily classic/tbc content. This is why trickerer added damage modifiers based on level bracket too. Experiment with those modifiers in the config and see what makes sense to you.

**Comment by Grimfeather** [comment]
I see. My server is set to vanilla content only atm.
So no TBC items. The other 2 modifiers are at default settings as well.

NpcBot.Mult.Damage.Physical = 1.0
NpcBot.Mult.Damage.Levels = 1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0

These brackets go from 0-9, 10-19, etc up to 70-79 and then 80+
so that might be it then. I'm in the 60-69 bracket, which is most likely based on TBC content.
I understand why the brackets are like this though,
if expansions are available players go to TBC at 60, heck even 58 sometimes.

what about the survivability? any idea how the warrior in the video went from 20 to 100% hp? (1 minute into the vid)
thanks for the reply. these 3 settings will let me tweak the damage.

**Comment by Russianranger** [comment]
They use potions on cooldown appropriate to their level. Most likely the warrior popped a potion. 

Also remember, you can set their max health and healing capabilities in the config. Again, there are a lot of levers you can pull to help customize them into what you feel appropriate. 

It's a lot easier to start off Overpowered instead of under powered and then give the tools to change later. Otherwise you get a lot of casual users commenting about how bots are too weak.

**Comment by Grimfeather** [comment]
I've looked into it more closely. There is more stuff going on.
First, I need to correct this, I missunderstood the brackets. 
I thought the bots strength was adjusted for each bracket by the mod,
but it's just a way for us to set their strength within those brackets. They're not weaker or stronger by default.
 
I'm testing as a hunter, at level 60. with a hunter npcbot next to me.
I gave the npcbot the exact same gear, same spec.

The hunter npcbot is doing WAY more damage and has a much higher crit rate and seems to use less mana
that's all with the default multiplier of 1.0
With some skills setting the multiplier to 0.6 is about right, the npcbots arcane shot damage will then be closer to mine
but for serpent sting it's closer to 0.8. so it's not consistent.

Plus the npcbots pet does all sorts of stuff it shouldn't be able to
example: we both have a wolf. and the npcbot's pet is casting thunderstomp and owl's focus
both are not available for a wolf that has the ferocity skill tree.

Maybe I'm just looking at all of this the wrong way.
I'm looking for realism, while it's just a fun way to let users group up and kill stuff, clear dungeons.
I'll close this issue.
